src/estimation/src/BerdyHelper.cpp

```diff
@@ -1366,6 +1387,48 @@ bool BerdyHelper::computeBerdySensorMatrices(SparseMatrix<iDynTree::ColumnMajor>
         // bY for the joint wrenches is zero
     }
 
+    ////////////////////////////////////////////////////////////////////////
+    ///// COM ACCELERATION SENSOR
+    ////////////////////////////////////////////////////////////////////////
+    if (m_options.includeCoMAccelerometerAsSensor)
+    {
+
+        // Get the row index corresponding to the com accelerometer sensor
+        // TODO: Correct the link index pssed here
+        IndexRange comAccelerometerRange = this->getRangeLinkSensorVariable(COM_ACCELEROMETER_SENSOR, 0);
+
+        for(LinkIndex idx = 0; idx < static_cast<LinkIndex>(m_model.getNrOfLinks()); idx++)
+        {
+            // Get the column index corresponding to the net link external wrench sensor
+            IndexRange netExternalWrenchSensor = this->getRangeLinkSensorVariable(NET_EXT_WRENCH_SENSOR, idx);
+
+            iDynTree::Rotation base_R_m_link = base_H_m_links.at(idx).getRotation();
+
+//            Matrix3x3 base_R_m_link_eigen_matrix;
```

Comment:
Comments to be removed.

If you would like to copy a Rotation matrix into a Matrix3x3, you could consider using,
``` cpp
#include <iDynTree/Core/EigenHelpers.h>
iDynTree::Matrix3x3 out;
auto in = iDynTree::Rotation::Identity();

iDynTree::toEigen(out) = iDynTree::toEigen(in);

```

Reply:
For more details on toEigen(), you could refer to https://github.com/robotology/idyntree/issues/510

Reply:
`iDynTree::Rotation base_R_m_link` was correctly handled in adding as a submatrix. In anycase, I updated the rotation matrix handling to be explicitly eigen matrix type. Updated in https://github.com/robotology/idyntree/pull/571/commits/6d9fac92d2c1fc56debff9ddea211d4a7162213b @prashanthr05 please check the code and resolve this comment